how-to-trust-fiddler-root-certificate.md

```diff
@@ -0,0 +1,39 @@
+---
+title: How to trust the Fiddler Root certificate
+description: Step by step tutorial for trusting the Fiddler root certificate
+type: how-to
+page_title: How to trust the Fiddler Root certificate
+slug: trust-fiddler-root-cert
+position: 100
+tags: fiddler, how-to
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+
+## Environment
+
+> **Service:** 3rd party tool
+> 
+> **Other stuff:** Other stuff here
+
+## Description
+When the Fiddler Root certificate on the machine is not trusted, you cannot capture HTTPS traffic and Fiddler logs do not contain any information about Telerik Platform/AppBuilder requests.
+
+This KB provides general steps for trusting the Fiddler Root certificate on your local machine. This is normally done automatically upon installing Fiddler, but certain environment configurations disallow this and you have to do it manually. 
```

Comment:
This article

upon installing -> when installing

Too many "this" in one paragraph (and one sentence)